nixos/doc/manual/from_md/release-notes/rl-2205.section.xml

```diff
@@ -2585,6 +2585,12 @@ cp /var/lib/redis/dump.rdb &quot;/var/lib/redis-mastodon/dump.rdb&quot;
           generating host-global NNCP configuration.
         </para>
       </listitem>
+      <listitem>
+        <para>
+          The <literal>nxt-python</literal> package was added at version
+          3.0.1.
+        </para>
+      </listitem>
```

Comment:
I don't think I've ever seen release notes being written because of a package being added ? 

I'm not opposed to it, but it's not something that's usually done

Reply:
I have no opinion obviously, it's my first PR. I'll go with you judgement. Should I remove it?